README.md

```diff
@@ -0,0 +1,33 @@
+# DamagingBlocks
+
+## Goal: 
+To allow modders to create blocks which cause damage to players upon physically touching them or destroy objects which land on the blocks (i.e. Lava)
+
+## Usage:
+
+The DamagingBlocks module uses a component and a system. Essentially, a component is a request for an entity to have a feature, and a system is provides instructions for the entity to perform the feature.
```

Comment:
Rather than explain here how Components and Systems work consider linking to the engine's wiki page on the topic since it covers it already: https://github.com/MovingBlocks/Terasology/wiki/Entity-System-Architecture

Instead you could briefly mention that to make the damage work you would attach the DamagingComponent to a prefab used for a block that should cause the damage, then the System will take care of applying the damage.

You could inline a brief example of a prefab that makes it work. Although I am just now realizing there might not be an obvious example of a prefab-based damage approach, all we have right now is the hard coded check for lava blocks, hmm. Up for making a new damaging block as well? :D @portokaliu could probably help! Can skip the prefab example for the time being if needed.